### PR TITLE
Doc: Add Unleash SDK for C++ to Community SDKs list.

### PR DESCRIPTION
**Pull request by aruizs**
Hello,

I'd like to add a reference of my SDK for C++ to the Community SDK list.

Thanks!

### CODE REVIEW

File: website/docs/sdks/index.md [hunk 1]
```diff
@@ -112,6 +112,7 @@ Here's some of the fantastic work our community has done to make Unleash work in
 - [silvercar/unleash-client-kotlin](https://github.com/silvercar/unleash-client-kotlin) (Kotlin)
 - [Stogon/unleash-bundle](https://git.stogon.io/Stogon/unleash-bundle/) (PHP - Symfony)
 - [uekoetter.dev/unleash-client-dart](https://pub.dev/packages/unleash) (Dart)
+- [aruizs/unleash-client-cpp](https://github.com/aruizs/unleash-client-cpp) (C++)
```
Comment (thomasheartman) [inline website/docs/sdks/index.md:115]: Sweet! Can you move this to between "AppsFlyer[...]" and "mikefrancis[...]"?

**Review by thomasheartman — CHANGES_REQUESTED** [review]
Hey, @aruizs 👋🏼  That's great; we'd love to put it on the list 😄  Just one request before we merge it: could move it up so that the list remains alphabetically sorted?

File: website/docs/sdks/index.md (hunk 1 above)
Comment (aruizs) [reply website/docs/sdks/index.md:115]: Sure! I didn't realize the list follows that order, I applied a FIFO method.

Comment (thomasheartman) [reply website/docs/sdks/index.md]: No worries! That looks great!

**Review by thomasheartman — APPROVED** [review]

**Pull request merged**